cmd/algons/dnsaddrCmd.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2019-2023 Algorand, Inc.
+// This file is part of go-algorand
+//
+// go-algorand is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// go-algorand is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	log "github.com/algorand/go-algorand/logging"
+	"github.com/algorand/go-algorand/network/p2p/dnsaddr"
+	"github.com/algorand/go-algorand/tools/network"
+)
+
+var (
+	dnsaddrDomain string
+	secure        bool
+)
+
+func init() {
+	dnsaddrCmd.AddCommand(dnsaddrTreeCmd)
+
+	dnsaddrTreeCmd.Flags().StringVarP(&dnsaddrDomain, "domain", "d", "", "Top level domain")
+	dnsaddrTreeCmd.MarkFlagRequired("domain")
+	dnsaddrTreeCmd.Flags().BoolVarP(&secure, "secure", "s", true, "Enable dnssec")
+}
+
+var dnsaddrCmd = &cobra.Command{
+	Use:   "dnsaddr",
+	Short: "Get, Set, and List Dnsaddr entries",
+	Long:  "Get, Set, and List Dnsaddr entries",
+	Run: func(cmd *cobra.Command, args []string) {
+		// Fall back
+		cmd.HelpFunc()(cmd, args)
+	},
+}
+
+var dnsaddrTreeCmd = &cobra.Command{
+	Use:   "tree",
+	Short: "Recursively resolves and lists the dnsaddr entries of the given domain",
+	Long:  "Recursively resolves and lists the dnsaddr entries of the given domain",
+	Run: func(cmd *cobra.Command, args []string) {
+		controller := dnsaddr.NewMultiaddrDNSResolveController(network.NewResolveController(secure, "127.0.0.1", log.Base()))
```

Comment:
It's the fallback dns resolver address. I think, as it's written it will just resolve to whatever is in `resolve.conf`, same as the system resolver does. We could default to `1.1.1.11` or `8.8.8.8` here if we want.

Reply:
Yeah I used that for two reasons...1 is as you said we use it currently at the hardcoded default, and 2. being that my dev OS seems to use it (as far as I understand)
```
DNS configuration (for scoped queries)

resolver #1
  search domain[0] : Home
  nameserver[0] : 127.0.0.1
```